docs/api/version-history.md

```diff
@@ -143,7 +143,7 @@ keywords: "API, Docker, rcli, REST, documentation"
 
 [Docker Engine API v1.27](https://docs.docker.com/engine/api/v1.27/) documentation
 
-* `GET /containers/(id or name)/stats` now includes an `online_cpus` field in both `precpu_stats` and `cpu_stats`. If this field is `nil` then for compatibility with older daemons the length of the corresponding `cpu_usage.percpu_usage` array should be used.
+* `GET /containers/(id or name)/stats` now includes an `online_cpus` field in both `percpu_stats` and `cpu_stats`. If this field is `nil` then for compatibility with older daemons the length of the corresponding `cpu_usage.percpu_usage` array should be used.
```

Comment:
I know this confused me a couple of times as well, but I think `precpu_..` is correct; https://github.com/moby/moby/blob/2c6a1e1878975550cd2e7a98494773903e39eaf7/api/types/stats.go#L168